ironfish/src/wallet/accounts.test.ts

```diff
@@ -341,4 +342,146 @@ describe('Accounts', () => {
       )
     })
   })
+
+  describe('getTransactionStatus', () => {
+    it('should show unconfirmed transactions as unconfirmed', async () => {
```

Comment:
Can you add a test to test the 'unknown' status

Reply:
I think that would be helpful -- I think we need to standardize transaction statuses across the CLI and our documentation